docs/source/interfaces/CrystalFieldPythonInterface.rst

```diff
@@ -351,5 +351,39 @@ Then call `fit()` method::
     fit.fit()
     
 After fitting finishes the `CrystalField` object updates automatically and contains new fitted parameter values.
+
+Finding Initial Parameters
+--------------------------
+
+If the initial values of the fitting parameters are not known they can be estimated using `monte_carlo()` method.
+It randomly searches the parameter space in a given region such that the calculated spectra are as close to the
+fit data as possible. The method uses :ref:`MonteCarloParameters <algm-MonteCarloParameters>` internally. See
```

Comment:
We just need to update this documentation reference with the new algorithm name and then I think this is good to go.